addon/templates/components/frost-list.hbs

```diff
@@ -14,11 +14,11 @@
       }}
     {{/if}}
 
-    {{#if (and pagination itemExpansion)}}
+    {{#if (and pagination (or itemExpansion isAnyTypedItemExpansion))}}
       <div class='frost-list-header-divider'></div>
     {{/if}}
 
-    {{#if itemExpansion}}
+    {{#if (or itemExpansion isAnyTypedItemExpansion)}}
```

Comment:
same comment ^

Reply:
I just addressed this by moving the logic here into a CP.